doc-content/enterprise-only/openshift/environment-authoring-con.adoc

```diff
@@ -1,16 +1,46 @@
-[id='environment-authoring-con']
+[id='environment-authoring-con'] 
+ifdef::PAM[]
 = Authoring environment
-You can deploy an environment for creating and modifying processes using {CENTRAL}. It consists of {CENTRAL} for the authoring work and {KIE_SERVER} for test execution of the processes.
+You can deploy an environment for creating and modifying processes using {CENTRAL}. It consists of {CENTRAL} for the authoring work and {KIE_SERVER} for test execution of the processes. If necessary, you can connect additional {KIE_SERVERS} to the {CENTRAL}.
+endif::PAM[]
+ifdef::DM[]
+= Authoring or managed server environment
+You can deploy an environment for creating and modifying services using {CENTRAL} and for running them in {KIE_SERVERS} managed by {CENTRAL}. This environment consists of {CENTRAL} and one or more {KIE_SERVERS}.
 
-Depending on your needs, you can deploy either a single authoring environment or a high-availability (HA) authoring environment.
+You can use {CENTRAL} both to develop services and to deploy them to {KIE_SERVERS}. You can connect several {KIE_SERVERS} to one {CENTRAL} to manage deployment of services to each of the servers. 
 
-A single authoring environment contains two pods. One of the pods runs {CENTRAL}, the other runs {KIE_SERVER}. The {KIE_SERVER} includes an embedded in-memory H2 database engine. This type of environment uses the least possible amount of resources. However, because of the in-memory database, restarting the {KIE_SERVER} pod leads to loss of all process information.
+If necessary, you can create separate environments, so that you can use one deployment of {CENTRAL} to author services (_authoring environment_) and another deployment of {CENTRAL} to manage deployment of staging or production services on several {KIE_SERVERS} (_managed server environment_). Usually, one {KIE_SERVER} is sufficient for a dedicated authoring environment. You can use an external Maven repository to store services from an authoring environment and deploy them to a separate managed server environment.
 
-An HA authoring environment contains several pods. Both {CENTRAL} and {KIE_SERVER} are provided in scalable pods that can run in parallel and share persistent storage. The database is provided by a separate pod. Use a high-availability authoring environment to provide maximum reliability and responsiveness, especially if several users are involved in authoring at the same time.
+For {PRODUCT}, the procedures to deploy an authoring environment and a managed server environment are the same. You must first deploy an authoring environment template, consisting of {CENTRAL} and one {KIE_SERVER}. Then, if necessary, you can deploy additional {KIE_SERVER} templates in the same namespace to create an environment  with multiple {KIE_SERVERS}.
```

Comment:
I think we should also mentioned there what is different between DM authoring env. and managed env.

Reply:
I deleted this difference on your review, I'll re-add.